### ISSUE TITLE
Monitored control in placeholder section

### ISSUE THREAD

**Issue by odi76** [bsgriggs/mendix-unsaved-changes-message#1]
Hi Benjamin,

Good work! I miss this capability in Mendix features very much.

I would like to apply this feature to a link (Button widget with 'link' render mode) placed into a placeholder section of the layout applied by the page. So far, without any success.

I tested the same Button in the Main section of the layout and it is working, but not in the placeholder section.

Could you help me what I miss? Or could this be a bug?

br,
Istvan

**Comment by bsgriggs** [comment]
Hi Istvan,
Did you give the button in the layout the Watching Class configured in the widget?

**Comment by odi76** [comment]
Yes, I followed the document. As I explained, the same building elements worked in the Main section of the layout.

**Comment by bsgriggs** [comment]
Do you have a test project you can provide that reproduces this issue? I'm afraid I don't quite understand the scenario.
